internal/ecrconsumer/sqs.go

```diff
@@ -0,0 +1,86 @@
+package ecrconsumer
+
+import (
+	"context"
+	"time"
+
+	"github.com/Wattpad/sqsconsumer"
+	"github.com/Wattpad/sqsconsumer/middleware"
+
+	// Metric Imports
+	kitlog "github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/metrics"
+	"github.com/go-kit/kit/metrics/dogstatsd"
+)
+
+// NOTE: Refactor to take Env struct
+
+type SQSConfig struct {
+	Name        string
+	Region      string
+	DatadogHost string
+	DatadogPort string
+}
+
+func NewSQSConfig(name string, env *Config) (*SQSConfig, error) {
+	return &SQSConfig{
+		Name:        name,
+		Region:      env.AWSRegion,
+		DatadogHost: env.DogstatsdHost,
+		DatadogPort: env.DogstatsdPort,
+	}, nil
+}
+
+func NewSQSConsumer(logger kitlog.Logger, dd *dogstatsd.Dogstatsd, hist metrics.Histogram, name string, svc sqsconsumer.SQSAPI) (*sqsconsumer.Consumer, error) {
```

Comment:
The `dd` client is unused and can be removed, the consumer only depends on the histogram.